### ISSUE TITLE
[PATH] Remake Ghostrider of Karabor path

### ISSUE THREAD

**Issue by aresbrutus** [Atlantiss/NetherwingBugtracker#2667]
http://db.hellfire-tbc.com/?npc=21784

**Comment by aresbrutus** [comment]
Fixed in rev `Castilla`

**Comment by Mikik3jr** [comment]
Shouldn't they patrol counter clockwise? They patrolled clockwise twice for me.
https://www.youtube.com/watch?v=ded05pOZyDI
![image](https://user-images.githubusercontent.com/43505031/48618808-c7231280-e99a-11e8-9a53-d304182f9f5b.png)
![image](https://user-images.githubusercontent.com/43505031/48618831-d6a25b80-e99a-11e8-9631-7bd51fb587f8.png)
![image](https://user-images.githubusercontent.com/43505031/48618870-eb7eef00-e99a-11e8-89b4-056afc6a0d83.png)
![image](https://user-images.githubusercontent.com/43505031/48618914-0c474480-e99b-11e8-9366-808e0e0f1408.png)
![image](https://user-images.githubusercontent.com/43505031/48618935-1c5f2400-e99b-11e8-8d80-4933ddc9422c.png)

**Comment by kwah8787** [comment]
I wasn't sure if I should make a new issue or put it here. I just did this quest to kill these fellas and I ran into some problems. Nothing major but I was reading wowhead comments and they said that splitting them (there's 4 of them) with distract was possible. It was not. After distract wore off or I engaged him he would out of combat run back to the other pack, ignoring me. Also if you run up and engage one normally, and the rest keep running, the one you engaged can bug out and run back to the pack as well. According to the wowhead comments they are not supposed to be linked but they are splittable. One other thing I ran into is when I was finally able to get them to stop, one of them fully disappeared during the fight but I don't know how you would reproduce that.
